frontend/templates/troubleshooting/Resource.tsx

```diff
@@ -1,20 +1,137 @@
-import { Stack, Image, Badge, StackProps, Link, Wrap } from '@chakra-ui/react';
+import {
+   Stack,
+   Image,
+   Badge,
+   StackProps,
+   Link,
+   Wrap,
+   HStack,
+   Text,
+   Box,
+   SystemProps,
+   Button,
+   ButtonProps,
+   ThemingProps,
+   useBreakpoint,
+} from '@chakra-ui/react';
 import { Guide } from './hooks/GuideModel';
 import { FaIcon } from '@ifixit/icons';
 import { faClock } from '@fortawesome/pro-solid-svg-icons';
 import Prerendered from './prerendered';
 import { DifficultyThemeLookup, GuideDifficultyNames } from './DifficultyBadge';
+import { Product } from '@models/product';
+import { useSelectedVariant } from '@templates/product/hooks/useSelectedVariant';
+import { useIsProductForSale } from '../product/hooks/useIsProductForSale';
+import { shouldShowProductRating } from '../../../packages/helpers/product-helpers';
```

Comment:
Is `/packages/` used enough to warrant an alias?

Reply:
This should have been from `@ifixit/helpers`. Fixed in [1ee69bc](https://github.com/iFixit/react-commerce/pull/1695/commits/1ee69bce3e16567f095ee7636cfcd4fd4d3ea3c5).